concent_api/core/subtask_helpers.py

```diff
@@ -70,6 +70,22 @@ def _update_timed_out_subtask(subtask: Subtask) -> None:
         )
     elif subtask.state == Subtask.SubtaskState.FORCING_ACCEPTANCE.name:  # pylint: disable=no-member
         task_to_compute = deserialize_message(subtask.task_to_compute.data.tobytes())
+        update_subtask_state(
+            subtask=subtask,
+            state=Subtask.SubtaskState.ACCEPTED.name,  # pylint: disable=no-member
+        )
+        store_pending_message(
+            response_type=PendingResponse.ResponseType.SubtaskResultsSettled,
+            client_public_key=subtask.provider.public_key_bytes,
+            queue=PendingResponse.Queue.Receive,
+            subtask=subtask,
+        )
+        store_pending_message(
+            response_type=PendingResponse.ResponseType.SubtaskResultsSettled,
+            client_public_key=subtask.requestor.public_key_bytes,
+            queue=PendingResponse.Queue.ReceiveOutOfBand,
+            subtask=subtask,
+        )
 
         def finalize_claim_for_acceptance_case() -> None:
```

Comment:
@rwrzesien you moved this section. Why? I don't say it is wrong, but I thought that there is no difference where you put `transaction.on_commit(....`. Does it have to be at the end of `if` or `elif`?

Reply:
You are right, now it doesn't make a difference, but I have moved it to make it more readable in a way of sentence of operation chain. But if you think it is not necessary I can drop this change.

Reply:
It is OK, thanks for explanation. I was just asking :)